test/Tests/src/Data/Ordering/Comparator_Spec.enso

```diff
@@ -24,8 +24,13 @@ type No_Ord
     Value number
 
 type No_Ord_Comparator
-    compare x y = Nothing
-    hash x = 0
+    compare x y =
```

Comment:
Very curious what is going on here.

Reply:
test classes with no comparator. This was the cause for some warnings in the test runs.

Reply:
We have an unused variable warning.

Ideally we shouldn't commit code that contains warnings, but since the CI only checks for Test suite errors and not warnings, it sometimes slips through.

Maybe we should introduce some env-var / parameter which could work like C++'s `-Werror` setting? We could enable it on CI which would ensure that we catch any warnings before merging.